packages/amplify_auth_cognito/test/amplify_auth_cognito_resetPassword_test.dart

```diff
@@ -46,9 +46,9 @@ void main() {
   });
 
   test('resetPassword request returns a ResetPasswordResult', () async {
-    expect(
-        await auth.resetPassword(
-            request: ResetPasswordRequest(username: 'testUser')),
-        isInstanceOf<ResetPasswordResult>());
+    var res = await auth.resetPassword(
+        request: ResetPasswordRequest(username: 'testUser'));
+    expect(res, isInstanceOf<ResetPasswordResult>());
```

Comment:
Nit: `isA`?

Reply:
Yep was just pointing it out because I thought I saw we made the change somewhere in this PR